src/flb_hash.c

```diff
@@ -41,7 +41,7 @@
  * 2. It will not produce the same results on little-endian and big-endian
  *    machines.
  */
-static unsigned int gen_hash(const void *key, int len)
+unsigned int gen_hash(const void *key, int len)
```

Comment:
public interfaces must be prefixed with flb_, e.g: ```flb_hash_gen_hash``` or ```flb_hash_generate```

Reply:
@edsiper Done!
- Rename `gen_hash` to `flb_hash_generate`.